### PR TITLE
change folder structure and change documentation (+ more)

### PR DESCRIPTION
**Pull request by wkloucek**
- changed the folder structure do have a `charts` subfolder
- reworked the readmes / documentation
- changed the license to apache 2 (like oCIS itself)
- added a warning on install when persistence is not enabled for critical services (NOTES.txt), therefore changed persistence defaults to disabled
- added missing configuration to values.yaml
- removed unnecessary config from the values.yaml (proxy tls setting, since really only make sense on the ingress)
- changed defaults to some more sane values
- added some tickets to this repo and referenced them in the code

### CODE REVIEW

File: charts/ocis/templates/nats/deployment.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
   selector:
     matchLabels:
         app: nats
-  replicas: 1 #TODO: scale out ?
+  replicas: 1 #TODO: scale out? https://github.com/owncloud/ocis-charts/issues/12
```
Comment (dragonchaser) [inline charts/ocis/templates/nats/deployment.yaml:12]: I know this is a tech preview, but IMHO Todos should not be in here.

File: charts/ocis/templates/settings/deployment.yaml [hunk 2]
```diff
@@ -66,7 +66,6 @@ spec:
           volumeMounts:
           - name: settings-data
             mountPath: /var/lib/ocis
-            # TODO: check why we loose roles after a settings restart
```
Comment (dragonchaser) [inline charts/ocis/templates/settings/deployment.yaml:69]: see previous comment.

**Review by dragonchaser — CHANGES_REQUESTED** [review]

File: charts/ocis/templates/nats/deployment.yaml (hunk 1 above)
Comment (wkloucek) [reply charts/ocis/templates/nats/deployment.yaml]: Unfortunately these more TODOs than this TODO, that are still to be done and to big / too much work for this PR. At least there is no more TODO without issue (link).

Would you prefer to have the TODO comments removed from the chart? I thought it's nice to have a direct link to the GitHub issue from the code.

Comment (dragonchaser) [reply charts/ocis/templates/nats/deployment.yaml]: Sounds good, so at least we have a reference for tracking.

**Review by dragonchaser — APPROVED** [review]

**Pull request merged**